proto/zitadel/policy/v2alpha/login_policy.proto

```diff
@@ -0,0 +1,111 @@
+        }
+    ];
+    string default_redirect_uri = 8 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines where the user will be redirected to if the login is started without app context (e.g. from mail)";
+            example: "\"https://acme.com/ui/console\"";
+        }
+    ];
+    google.protobuf.Duration password_check_lifetime = 9 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"864000s\"";
+        }
+    ];
+    google.protobuf.Duration external_login_check_lifetime = 10 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"864000s\"";
+        }
+    ];
+    google.protobuf.Duration mfa_init_skip_lifetime = 11 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"2592000s\"";
+        }
+    ];
+    google.protobuf.Duration second_factor_check_lifetime = 12 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"64800s\"";
+        }
+    ];
+    google.protobuf.Duration multi_factor_check_lifetime = 13 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            example: "\"43200s\"";
+        }
+    ];
+    repeated SecondFactorType second_factors = 14;
+    repeated MultiFactorType multi_factors = 15;
+    // If set to true, the suffix (@domain.com) of an unknown username input on the login screen will be matched against the org domains and will redirect to the registration of that organization on success.
+    bool allow_domain_discovery = 16 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "If set to true, the suffix (@domain.com) of an unknown username input on the login screen will be matched against the org domains and will redirect to the registration of that organization on success."
+        }
+    ];
+    bool disable_login_with_email = 17 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the user can additionally (to the login name) be identified by their verified email address"
+        }
+    ];
+    bool disable_login_with_phone = 18 [
+        (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+            description: "defines if the user can additionally (to the login name) be identified by their verified phone number"
+        }
+    ];
+}
+
+enum SecondFactorType {
+    SECOND_FACTOR_TYPE_UNSPECIFIED = 0;
+    SECOND_FACTOR_TYPE_OTP = 1;
+    SECOND_FACTOR_TYPE_U2F = 2;
+}
+
+enum MultiFactorType {
+    MULTI_FACTOR_TYPE_UNSPECIFIED = 0;
+    MULTI_FACTOR_TYPE_U2F_WITH_VERIFICATION = 1;
+}
+
+enum PasswordlessType {
+    PASSWORDLESS_TYPE_NOT_ALLOWED = 0;
+    PASSWORDLESS_TYPE_ALLOWED = 1;
+}
```

Comment:
do we want to rename it to passkeys?

Reply:
can be renamed if necessary, no idea if there are later more names to login "without password"

Reply:
@hifabienne what's your opinion on this?